bin/main.rb

```diff
@@ -1,62 +1,21 @@
 #!/usr/bin/env ruby
 
 require 'colorize'
-
-class User
-  attr_reader :symb, :name
-  def initialize(symb, name)
-    @name = name
-    @symb = symb
-  end
-end
-
-class Board
-  attr_reader :symb, :counter, :status, :b
-
-  def initialize
-    @b = Array.new(3) { |i| Array.new(3) { |j| ((i * 3 + j + 1)).to_s } }
-    @counter = 9
-    @symb = ''
-  end
-
-  def turn(symb, loc)
-    if !loc.to_s.match?(/[1-9]/) || loc.to_s.length != 1 || @b[(loc - 1) / 3][(loc - 1) % 3].match?(/[XO]/)
-      @status = false
-    else
-      @status = true
-      @b[(loc - 1) / 3][(loc - 1) % 3] = symb
-      if winner(@b)
-        @counter = 0
-        @symb = symb
-      end
-      @counter -= 1
-    end
-  end
-
-  private
-
-  def winner(test)
-    trans = test.transpose
-    (0...test.size).collect { |i| test[i][i] }.uniq.count == 1 ||
-      (0...test.size).collect { |i| test[test.size - i - 1][i] }.uniq.count == 1 ||
-      test.map { |x| x.uniq.count }.include?(1) ||
-      trans.map { |x| x.uniq.count }.include?(1)
-  end
-end
+require_relative '../lib/user.rb'
+require_relative '../lib/board.rb'
 
 puts '*********************************'.red
 puts '****'.red + '      TIC TAC TOE        ' + '****'.red
 puts '*********************************'.red
 user = []
-user_symbols = %w[X O]
+board = Board.new
 2.times do |i|
   print "User #{i + 1}: Please write your name: "
   name = gets.chomp
-  user << User.new(user_symbols[i], name)
+  user << User.new(board.symbols[i], name)
   puts "Welcome #{user[i].name.yellow}. your symbol is #{user[i].symb.yellow}."
 end
 
-board = Board.new
 3.times { |i| p board.b[i] }
```

Comment:
That chunk of code appears many time.
Write a method into the `Board` class that will return a **String** which will be print in `main.rb`
```rb
class Board
  ...
  def display
    " ... "
  end
  ...
end
```

Call it like this in `bin/main.rb`
```rb
p board.display
```